Firestore/Protos/build-protos.sh

```diff
@@ -27,24 +27,38 @@ pod update
   --nanopb_out="--options-file=protos/%s.options:nanopb" \
   `find protos -name *.proto -print | xargs`
 
-# Remove "well-known" protos from objc. (We get these for free. We only need
-# them for nanopb.)
+# Remove "well-known" protos from objc and cpp. (We get these for free. We only
+# need them for nanopb.)
 rm -rf objc/google/protobuf/
+rm -rf cpp/google/protobuf
 
 # If a proto uses a field named 'delete', nanopb happily uses that in the
 # message definition. Works fine for C; not so much for C++. Rename uses of this
 # to delete_ (which is how protoc does it for c++ files.)
 perl -i -pe 's/\bdelete\b/delete_/g' `find nanopb -type f`
 
+# Rename nanopb's headers from foo.pb.h to foo.nanopb.h. This avoids collisions
+# with libprotobuf. (This mostly doesn't matter, as it's odd to want to use
+# both. However, we do use both in the test suite to verify our nanopb based
+# serializer is compatible with an equivalent libprotobuf implementation.)
+for f in $(find nanopb -name \*.pb.h -or -name \*.nanopb.c); do
```

Comment:
Why `-or -name \*.nanopb.c`? Would any files match that? Also your move would generate `foo.nanopb.c.nanopb.h` which you don't want anyway.

If you wanted to rename the `.c` files too, it would look something like this:
```bash
for f in $(find nanopb -name "*.pb.*"); do
  mv "$f" "${f/.pb./.nanopb.}"
done
```

Reply:
Oops. No, I didn't meant to include that. (No, no files would match.)

(I was originally considering moving the .c files too, but decided against it. This is probably some artifact of that, but where the 'nanopb' portion comes from...?)

Removed '-or ...'